java/mapreduce/src/java/org/apache/orc/mapred/OrcOutputFormat.java

```diff
@@ -75,6 +75,8 @@ public RecordWriter<NullWritable, V> getRecordWriter(FileSystem fileSystem,
     Path path = getTaskOutputPath(conf, name);
     Writer writer = OrcFile.createWriter(path,
         buildOptions(conf).fileSystem(fileSystem));
-    return new OrcMapredRecordWriter<>(writer);
+    return new OrcMapredRecordWriter<>(writer,
+        OrcConf.ROW_BATCH_SIZE.getInt(conf),
```

Comment:
https://github.com/apache/orc/pull/1108#discussion_r869544673

Should we split into two configurations? In scenarios with read and write orc, the same configuration should not be mixed.